pkg/authorization/api/validation/validation.go

```diff
@@ -292,7 +292,7 @@ func validateRoleBindingSubject(subject kapi.ObjectReference, isNamespaced bool,
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("name"), subject.Name, reason))
 		}
 		if !isNamespaced && len(subject.Namespace) == 0 {
-			allErrs = append(allErrs, field.Required(fldPath.Child("namespace"), ""))
+			allErrs = append(allErrs, field.Required(fldPath.Child("namespace"), "ServiceAccount subjects for ClusterRoleBindings must have a namespace"))
```

Comment:
This is targeted for humans - we typically use non-Kind form ("service account") in messages intended for end users